src/services/structures/structures.class.js

```diff
@@ -19,6 +19,43 @@ exports.Structures = class Structures extends Service {
       this.Model = db.collection('structures');
     });
 
+    app.post('/structures/:id/preSelectionner/:conseillerId', async (req, res) => {
+      let structureId = null;
+      let conseillerId = null;
+      try {
+        structureId = new ObjectID(req.params.id);
+        conseillerId = new ObjectID(req.params.conseillerId);
+      } catch (e) {
+        res.status(404).send(new NotFound('Structure not found', {
```

Comment:
@loicguillois Structure or conseiller not found ? (avec en params structureId et conseillerId)